README.md

```diff
@@ -8,8 +8,17 @@ This repository provides plugins and tools for integrating spatial audio into yo
 # Getting started with Spatial Audio for Unity
 Cloning this repository is not required to start using the Microsoft Spatializer in your Unity project. Visit the [documentation](https://docs.microsoft.com/en-us/windows/mixed-reality/spatial-sound-in-unity) for instructions on integrating the Microsoft Spatializer into your Unity project. For a more in-depth exploration of spatial audio, check out the [learning module](https://docs.microsoft.com/en-us/windows/mixed-reality/unity-spatial-audio-ch1). If you'd like to build the plugin yourself, see below.
 
-#### Legacy HoloLens 2 Spatializer plugin
-While it is recommended to switch over to the latest cross-platform spatializer plugin, the previous [HoloLens 2 specific spatializer plugin version](https://github.com/microsoft/spatialaudio-unity/tree/v1.0.246) with hardware offload suppport, remains available on [GitHub releases](https://github.com/microsoft/spatialaudio-unity/releases/tag/v1.0.246) and on a UPM feed via (Mixed Reality Feature Tool)[https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/configuration/usingupm?view=mrtkunity-2021-05]. 
+### Choosing the right spatializer 
+With requirements and features evolving over time, there are now 3 different Unity spatializer plugins available from Microsoft. Here's a brief description of their differences which can help decide the right plugin for a project.
+
+#### [Microsoft Spatializer v2](https://github.com/microsoft/spatialaudio-unity/releases/tag/v2.0.30-prerelease)
+This is the latest highly optimized cross-platform spatializer plugin for Windows and Android built from this repository. Although this plugin is currenly in the pre-release phase, it's being actively developed and recommended for any new projects, especially those that need to support both Windows and Android. This plugin uses the latest DSP engine that is highly optimized for both memory and CPU and fits well into Unity audio engine making it easier when supporting audio design features such as environmental reverb for the project's spatial audio mix.     

```

Comment:
I thought that the comment about "supporting audio design features such as environmental reverb" didn't make sense to me until I read the v1 spatializer text where the situation of not bringing the signal back to the host was described.  I'm not sure how to introduce that concept in the v2 section - maybe just omitting that phrase would simplify things.